python/ray/tune/suggest/bayesopt.py

```diff
@@ -102,6 +106,8 @@ def on_trial_complete(self,
                           error=False,
                           early_terminated=False):
         """Passes the result to BayesOpt unless early terminated or errored"""
+        if early_terminated and self.use_early_terminated_trials is False:
+            return
         if result:
```

Comment:
this should probably be like; 

```
if result:
	if early_terminated and self.use_early_terminated_trials is False:
        return
    ...
```

Can you also do this for all of the search algorithms?

Thanks!